### PR TITLE
Calendar Module: Add sample data

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**

### CODE REVIEW

File: modules/calendar.xml [hunk 1]
```diff
@@ -128,5 +128,35 @@ if(typeof ongoingEvent != 'undefined' && ongoingEvent && properties.currentEvent
 // Start effects for this widget
 $(target).xiboLayoutAnimate(properties);
     ]]></onVisible>
+    <sampleData><![CDATA[
+[{
+    "summary": "Sample 1",
+    "description": "Something here",
```
Comment (dasgarner) [inline modules/calendar.xml:134]: Can we have a go at making some better summaries/descriptions for these events? Even if its just "Sample Event 1" and the description explaining what we're doing with the dates.

**Review by dasgarner — COMMENTED** [review]
Looks good to me - I had one general comment, but will let Mauro review the code more closely. :+1:

**Review by maurofmferrao — APPROVED** [review]

**Review by maurofmferrao — APPROVED** [review]
Looks good!

File: modules/calendar.xml (hunk 1 above)
Comment (dasgarner) [reply modules/calendar.xml]: It looks great, please go ahead!! :+1:

**Pull request merged**